fs_attachment/models/ir_attachment.py

```diff
@@ -47,11 +47,6 @@ def clean_fs(files):
                 _logger.info(
                     "_file_delete could not unlink %s", full_path, exc_info=True
                 )
-            except OSError:
-                # Harmless and needed for race conditions
-                _logger.info(
-                    "_file_delete could not unlink %s", full_path, exc_info=True
-                )
```

Comment:
- From python3.3, IOError was merge into OSError: [https://docs.python.org/3/library/exceptions.html#OSError](https://docs.python.org/3/library/exceptions.html#OSError)
- So, pre-commit command replaced `except IOError: `by `except OSError:` at the line 50.